packages/langium/src/lsp/reference-finder.ts

```diff
@@ -28,6 +28,11 @@ export interface ReferenceFinder {
     findReferences(document: LangiumDocument, params: ReferenceParams, cancelToken?: CancellationToken): MaybePromise<Location[]>;
 }
 
+export interface ReferenceLocation {
```

Comment:
You only need this intermediate interface because your URIs are not compatible. Is it possible to use the one from VSCode? Or is there a reason?

Reply:
Afaik, the VSCode URI doesn't have information about the range